Sessions/session1.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html>

```

Comment:
Some of the parts of HTML design are not implemented in page. Please try to create all the sections as per the assignment image.

Reply:
done